Conceptpower+Spring/src/main/java/edu/asu/conceptpower/web/ConceptEditController.java

```diff
@@ -180,11 +146,39 @@ public String confirmlEdit(@PathVariable("id") String id,
 	 * @return The list of existing concepts
 	 */
 	@RequestMapping(method = RequestMethod.GET, value = "conceptEditSynonymView")
-	public @ResponseBody
-	ConceptEntry[] searchConcept(@RequestParam("synonymname") String synonymname) {
-		ConceptEntry[] entries = conceptManager
-				.getConceptListEntriesForWord(synonymname.trim());
-		return entries;
+	public @ResponseBody ResponseEntity<String> searchConcept(@RequestParam("synonymname") String synonymname) {

```

Comment:
I don't think you need the ResponseBody annotation if you return a ResponseEntity. But I'm not sure.